packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -354,6 +354,7 @@ mixin AutomaticKeepAliveClientMixin<T extends StatefulWidget> on State<T> {
 
   void _releaseKeepAlive() {
     _keepAliveHandle!.release();
+    _keepAliveHandle!.dispose();
```

Comment:
consider adding a comment that dispose and release do not imply each other. Alternatively, consider renaming release to something like notify.

Reply:
It seems renaming the method 'release' will introduce breaking change. Added comment.